src/pages/gateway/create-mesh.md

```diff
@@ -70,21 +57,94 @@ description: Create a configuration file for your mesh, access the gateway, and
     aio api-mesh:create mesh.json
     ```
 
+   **NOTE:** When creating or updating a mesh, the file to upload must have the `.json` filename extension.
```

Comment:
@keharper i can't get inline alerts to work inside of ordered lists. unindented, they break numbering. indented, the text doesn't display inside the text box. just fyi, since moving it resolves this